applications_tests/gls_sharp_navier_stokes_2d/couette2d_gls.output

```diff
@@ -25,8 +25,8 @@ Steady iteration :        3/3
    Number of degrees of freedom: 49923
    Volume of triangulation:      4
 div u : 1.86818e-07
-L2 error velocity : 0.000303581
-cells error_velocity  error_pressure 
- 1024 4.5262e-03    - 0            - 
- 4096 1.1934e-03 1.92 0          nan 
-16384 3.0358e-04 1.97 0          nan 
+L2 error velocity : 0.000303581 L2 error pressure: 0.00733328
+cells  error_velocity     error_pressure   
+ 1024 4.526187e-03    - 1.219155e-02     - 
+ 4096 1.193398e-03 1.92 6.854959e-03  0.83 
+16384 3.035815e-04 1.97 7.333278e-03 -0.10 
```

Comment:
Is it normal that pressure is not converging here? Because of your pressure imposition node?